config_static/jukebox.json

```
@@ -8469,5 +8469,45 @@
 "lobby": false,
 "jukebox": true,
 "genre": "Rock"
-}
+},
+{
+"url": "https://files.catbox.moe/vkx7jp.mp3",
+"title": "First of the Gang to Die",
+"duration": 2180,
+"artist": "Morrissey",
+"secret": false,
+"lobby": false,
+"jukebox": true,
+"genre": "Rock"
+},
+{
+"url": "https://files.catbox.moe/zvw9zg.mp3",
+"title": "A Day in the Life",
+"duration": 3120,
+"artist": "The Beatles",
+"secret": false,
+"lobby": false,
+"jukebox": true,
+"genre": "Rock"
+},
+{
+"url": "https://files.catbox.moe/jggdzg.mp3",
+"title": "Kid Dynamo",
+"duration": 2150,
+"artist": "The Buggles",
+"secret": false,
+"lobby": false,
+"jukebox": true,
+"genre": "Rock"
+},
+{
+"url": "https://files.catbox.moe/1dk4e4.mp3",
+"title": "Cold Cold Cold",
+"duration": 2160,
+"artist": "Cage the Elephant",
+"secret": false,
+"lobby": false,
+"jukebox": true,
+"genre": "Rock"
+},
```

you need to get rid of the ,
json expects another song elsewise
```suggestion
}
```